Sources/Datadog/RUMMonitor.swift

```diff
@@ -307,11 +307,13 @@ public class RUMMonitor: DDRUMMonitor, RUMCommandSubscriber {
             }
             return nil
         }()
-        addError(message: message, stack: stack, source: RUMInternalErrorSource(source), attributes: attributes)
+        // Note for CR: should `type` be nil?
```

Comment:
I think this is what we concluded with Android SDK team, right?
```suggestion
```

Reply:
yes, removed ✅